torch/_inductor/kernel/flex_attention.py

```diff
@@ -1541,7 +1541,7 @@ def bwd_dq_block_mn(
 
     if not IS_FULL_BLOCKS:
         if CHECK_BLOCK_BOUNDARY:
-            mask_mod_output = tl.where(offs_n2[None, :] < KV_LEN, mask_mod_output, float("-inf"))
+            mask_mod_output = tl.where(offs_n2[None, :] < KV_LEN, mask_mod_output, False)
```

Comment:
Also nit lets peel this off into early Stack so we can get this in and I can stop all those triton errors

Reply:
Hopefully I will land this PR soon haha.